distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -175,28 +183,86 @@ public void testCswRegistryCreateDup() throws Exception {
                 .post(CSW_PATH.getUrl());
     }
 
-    @Ignore
     @Test
     public void testCswRegistryStoreCreate() throws Exception {
-        //Stub test, waiting for additional features/service before implementing
-        //Use federation admin to create a registry entry
-        //on a 'remote' (aka loopback) catalog store
+        org.codice.ddf.security.common.Security.runAsAdmin(() -> {
```

Comment:
You shouldn't need this here. You're already using basic auth to login as admin. This would just be attaching a Java subject to the thread, which will get lost anyways during the transaction.